allennlp/tests/modules/stacked_bidirectional_lstm_test.py

```diff
@@ -55,3 +57,43 @@ def test_stacked_bidirectional_lstm_can_complete_forward_pass_seq2vec(self):
         mask = torch.ones(4, 5)
         output = encoder(input_tensor, mask)
         assert output.detach().numpy().shape == (4, 18)
+
+    @flaky(max_runs=3)
+    @pytest.mark.parametrize("dropout_name", ('layer_dropout_probability',
```

Comment:
Are these actually flaky? They shouldn't be, with a dropout rate of 0.9.

Reply:
Yes, I was surprised as well, they pass on average 4 out of 5 runs. 
I have had a look at the output tensors and the differences between them is marginal. 

The layer wise dropout is not applied to the last layer nor is the recurrent dropout in the augmented LSTM applied to the last hidden state that is outputted in the final states, so that might explain why they are so similar? Or I might have made a mistake in the code somewhere but I cannot find it.

Reply:
Oh I think maybe it's because you have all zero initial states? Can you try making them random so that we can rule that out.

Reply:
Changing the Initialisation value from 1 to 0.5 as well as changing the initial states solved the problem. Thanks for the suggestion. I have removed flaky now from the **stacked_bidirectional_lstm_test** as well as the **augmented_lstm_test**, as the augmented lstm had the same problem.